programs/ambassadors/ambassadors-program.md

```diff
@@ -0,0 +1,62 @@
+
+### How does the program work?
+
+- The ambassador selection process will be as transparent as possible.
+- The model currently in place for adding members to the Technical Steering Committee through PR will be used.
+- The TSC or delegated maintainers of community repositories will evaluate applications according to compliance requirements via a traditional PR flow.
+- All ambassador information will be displayed on the JSON Schema website in a list after the PR is merged.
+
+### Ambassador requirements
+
+The participation will be determined and checked annually. The minimum requirement for being an ambassador is to make **4 contributions per year** in the form of articles, talks, videos, podcasts, presentations, driving initiatives, etc.
+
+Ambassadors can be invited to review content (i.e., articles, talks, videos) related to the community. These reviews are optional and will be counted as an additional contribution.
+
+Some clarifications on the type of contributions:
+    
+- Articles, videos, and podcasts can be published on [json-schema.org/blog](https://json-schema.org/blog) or other domains. 
+- Talks and presentations can be displayed at different events or workshops. Within them, JSON Schema should appear at the core of the presentation. The presentation should not be being used as a disguised marketing tool for other products/projects.
+- Special contributions should be driven to support the community or enhance its visibility (i.e., getting new sponsors or driving initiatives) 
+- If we do not notice or accidentally miscount a contribution, just let us know.
+         
+Examples: 
+
+1. If contributors publish 3 articles and propel 1 initiative about JSON Schema in eight months, they will become ambassadors for the whole year.
+2. If contributors make 1 presentation, write 2 articles, and propel 1 initiative, they will become ambassadors for the whole year.
+3. If you were accepted as ambassador on the 5th of May 2023, you have already fulfilled the requirements. On the 5th of May 2024, we will check if you qualify again after the 5th of May 2023.
+   
+### Ambassadors duties
+
+- Be in tune with JSON Schema's mission and values.
+- Always respect the [code of conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md).
+- Be active in your role as an ambassador.
+
+### Ambassador benefits
+
+- Invitation to [the JSON Schema organization](https://github.com/orgs/json-schema-org/people).
+- Receive a special swag pack for Ambassadors (Swag availability date to be confirmed).
+- Receive additional swag for conference or workshop participants (Swag availability date to be confirmed).
+- Community-wide recognition.
+- All our respect for your contributions!
+
+### Become an JSON Schema Ambassador
+
+The process of becoming an ambassador is very simple. Go to the **community** repository and open a PR to edit the `ambassadors.json` file. The TSC will then evaluate your candidacy!
```

Comment:
What if we say the TSC AND existing Ambassadors?

Reply:
Sounds great